index.js

```diff
@@ -1487,13 +1487,13 @@
       [s(b, c), s(a, b), s(a, c)],
       Z.compose);
 
-  //# pipe :: [(a -> b), (b -> c), ..., (m -> n)] -> a -> n
+  //# pipe :: Foldable f => f (Any -> Any) -> a -> b
   //.
   //. Takes an array of functions assumed to be unary and a value of any type,
   //. and returns the result of applying the sequence of transformations to
   //. the initial value.
```

Comment:
I *think* I'm happy to continue to refer to _an array of functions_ because replacing “array” with “structure” is awkward.